ammico/cropposts.py

```diff
@@ -10,6 +10,16 @@
 
 # use this function to visualize the matches
 def plot_matches(img1, img2, keypoints1, keypoints2):
+    """

```

Comment:
No need to include documentation for this module as I have changed it quite a bit in my PR and will only lead to merge conflicts.